README.md

```diff
@@ -0,0 +1,36 @@
+Social Media Client 
```

Comment:
Well done on adding information about how to set up the app locally. However, some info is still missing. Info such as the badges, test scripts, etc. The info is a requirement for this delivery. Please, add them ASAP